### PR TITLE
Front End 

### PR DESCRIPTION
**Pull request by Hydroelectric29**
Added new front end components.  Wrote errors components and pod components. 
If you haven’t added my repo as a remote repo, run git remote add rob https://github.com/Hydroelectric29/Kubiquity.git
Create a new branch and pull my work: git checkout -b errors-nodes-fe && git pull rob main. 
Run npm install && npm run watch
In second terminal, run npm start
Make sure app loads; make sure you can see the Errors / Error and Nodes / Node components with the mock data.
We have our GET requests commented out because FE is not connected to BE. 
Styling is not included in this PR.
Data is currently mocked .  Based on what we might use in production.  Fetch request inactive.

### CODE REVIEW

**Comment by dlande000** [comment]
@Hydroelectric29 @JefZheng @davidzhangnyc one thing I just noticed from looking at this PR is that all our files should have new lines at the end, and all our tabs should be two spaces, not four. 

https://stackoverflow.com/questions/29972396/how-can-i-customize-the-tab-to-space-conversion-factor
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

**Review by dlande000 — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+
+// TODO: after MVP, convert React Class Components to Hooks.
+// TODO: after MVP, try out Typescript.
+
 const App = () => (
   <div>
-    React work will go here.
+    <h1>Kubiquity</h1>
+    <Errors />
+    <Pods/>
```
Comment (JefZheng) [inline src/App.jsx:13]: minor styling, add space after Pods

File: src/components/errors/Error.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                <div>Time of error: {createdAt}</div>
+                <div>Level of severity: {severity}</div>
+                <div>Recommended actions: {recommendedActions}</div>
+                <div>Pod: {pod}</div>
+            </>
+        )
+    }
+}
+
+export default Error;
```
Comment (JefZheng) [inline src/components/errors/Error.jsx:28]: Add new line at EOF

File: src/components/errors/Errors.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        return (
+            <div>
+                Errors
+                {errors}
+            </div>
+        )
+    }
+}
+
+export default Errors;
```
Comment (JefZheng) [inline src/components/errors/Errors.jsx:90]: Add new line at EOF

File: src/components/pods/Pod.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <div>ID: {podId}</div>
+        <div>Name: {name}</div>
+        <div>Current Memory Useage: {currentMemoryUsage}</div>
+        <div>Parent Node ID: {parentNode}</div>
+      </>
+    );
+  }
+}
+
+export default Pod;
```
Comment (JefZheng) [inline src/components/pods/Pod.jsx:24]: Add new line at EOF

File: src/components/pods/Pods.jsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        return (
+            <div>
+                Pods
+                {pods}
+            </div>
+        );
+    }
+}
+
+export default Pods;
```
Comment (JefZheng) [inline src/components/pods/Pods.jsx:85]: Add new line at EOF

**Review by JefZheng — APPROVED** [review]
Overall just some minor styling changes needed. If you run eslint, it should fix the styling. Good PR!

**Comment by dlande000** [comment]
> Overall just some minor styling changes needed. If you run eslint, it should fix the styling. Good PR!

These minor style changes will get fixed in the next FE PR, either from me or Rob.

File: src/components/errors/Error.jsx (hunk 2 above)
Comment (dlande000) [reply src/components/errors/Error.jsx:28]: Addressed in my comment above—everyone should change their VS Code to automatically add in new lines at EOF.

**Review by davidzhangnyc — APPROVED** [review]
LGTM

**Pull request merged**